test/suites/dev-tanssi-relay/ethereum-token-transfers/test_transfer_native_token.ts

```diff
@@ -267,7 +265,7 @@ describeSuite({
                 // Check balances were updated correctly.
                 const {
                     data: { free: sovereignBalanceAfter },
-                } = await context.polkadotJs().query.system.account(sovereign);
+                } = await context.polkadotJs().query.system.account(sovereignAccountEncoded);
```

Comment:
do we really need the encoded version here?

Reply:
No, confirmed that we don't need it.